src/markspelling.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, pwl, language='en_GB', errortotalprev=0):
         self.regexhtmlclean = re.compile(r'\`.+?\`')
         self.regexlink = re.compile(r'\[(.+?)\]\(.+?\)')
         self.regexurl = re.compile(r'https?://\S+')
+        self.regexlinkid = re.compile(r'\[\S+\]:\s[^{\s]*')
+        self.regexlinkparam = re.compile(r'{:\S+="\S+"}')
+        self.regexliquidtag = re.compile(r'{%.+%}')
```

Comment:
Do you only need to catch Liquid tags? What about objects (`{{ }}`)?

Reply:
Good point, added.